alf/environments/suite_dmc.py

```diff
@@ -11,121 +11,90 @@
+         image_size=100,
+         env_id=None,
+         discount=1.0,
+         visualize_reward=False,
+         max_episode_steps=1000,
+         control_timestep=None,
+         gym_env_wrappers=(),
+         alf_env_wrappers=()):
+    """ Load a MuJoCo environment.
 
-    This loader will not take environment_name, instead please use domain_name and tesk_name.
-    For installation of dmc2gym, see https://github.com/denisyarats/dmc2gym.
     For installation of DMControl, see https://github.com/deepmind/dm_control.
     For installation of MuJoCo210, see https://mujoco.org.
 
     Args:
-        environment_name (str): Do not use this arg, this arg is here to
-            match up with create_environment.
-        domain_name (str): The name of MuJoCo domain that is used.
-        task_name (str): The name of task we want the agent to do in the
-            current MuJoCo domain.
-        seed (int): Random seed for the environment. This parameter is
-            here to serve as a place holder and will be override by the
-            seed of create_environment().
+        environment_name (str): this string must have the format
+            "domain_name:task_name", where "domain_name" is defined by DM control as
+            the physical model name, and "task_name" is an instance of the model
+            with a parcular MDP structure.
         from_pixels (boolean): Output image if set to True.
-        image_size (int): The height and width of the output 
+        image_size (int): The height and width of the output
             image from the environment.
-        action_repeat (int): Action repeat of gym environment.
         env_id (int): (optional) ID of the environment.
         discount (float): Discount to use for the environment.
+        visualize_reward: if True, then the rendered frame will have
+            a highlighted color when the agent achieves a reward.
         max_episode_steps (int): The maximum episode step in the environment.
-            Note that the episode length in the alf will be max_episode_steps/action_repeat.
+        control_timestep (float): the time duration between two agent actions. If
+            this is greater than the agent's primitive physics timestep, then
+            multiple physics simulation steps might be performed between two actions.
+            The difference between multi-physics steps and "action repeats"/FrameSkip
+            is that the intermediate physics step won't need to render an observation
+            (which might save time if rendering is costly). However, this also
+            means that unlike "action repeats"/FrameSkip which accumulates rewards
+            of several repeated steps, only a single-step reward is obtained after
+            all the physics simulation steps are done. The total number of
+            physics simulation steps in an episode is
+            ``control_timestep / physics_timestep * frame_skip * max_episode_steps``.
```

Comment:
where is the parameter physics_timestep?

Reply:
> where is the parameter physics_timestep?

It's not specified here, but hardcoded for every mujoco xml file.